application/account-management/WebApp/shared/translations/locale/nl-NL.po

```diff
@@ -0,0 +1,281 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-10-07 23:40+0200\n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: nl-NL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: @lingui/cli\n"
+
+msgid "Account"
+msgstr "Account"
+
+msgid "Account settings"
+msgstr "Accountinstellingen"
+
+msgid "Account Settings"
+msgstr "Accountinstellingen"
+
+msgid "Actions"
+msgstr "Acties"
+
+msgid "Active Users"
+msgstr "Actieve gebruikers"
+
+msgid "Active users in the past 30 days"
+msgstr "Actieve gebruikers in de afgelopen 30 dagen"
+
+msgid "Add more in the Users menu"
+msgstr "Meer toevoegen via het gebruikersmenu"
+
+msgid "Added"
+msgstr "Toegevoegd"
+
+msgid "All Users"
+msgstr "Alle gebruikers"
+
+msgid "An error occurred while processing your request. {0}"
+msgstr "Er is een fout opgetreden bij het verwerken van uw verzoek. {0}"
+
+msgid "By continuing, you accept our policies"
+msgstr "Door verder te gaan, accepteer je onze voorwaarden"
+
+msgid "Can't find your code? Check your spam folder."
+msgstr "Kun je je code niet vinden? Controleer je spammap."
+
+msgid "Cancel"
+msgstr "Annuleren"
+
+msgid "Continue"
+msgstr "Doorgaan"
+
+msgid "Create your account"
+msgstr "Maak je account aan"
+
+msgid "Danger zone"
+msgstr "Gevaarzone"
+
+msgid "Delete"
+msgstr "Verwijderen"
+
+msgid "Delete Account"
+msgstr "Account verwijderen"
+
+msgid "Deleting the account and all associated data. This action cannot be undone, so please proceed with caution."
+msgstr "Het account en alle bijbehorende gegevens worden verwijderd. Deze actie kan niet ongedaan worden gemaakt, dus ga voorzichtig te werk."
+
+msgid "Didn't receive the code? Resend"
+msgstr "Code niet ontvangen? Verstuur opnieuw"
+
+msgid "Do you already have an account?"
+msgstr "Heb je al een account?"
+
+msgid "Domain"
+msgstr "Domein"
+
+msgid "Don't have an account? <0>Create one</0>"
+msgstr "Heb je geen account? <0>Maak er één aan</0>"
+
+msgid "E.g., CompanyX"
+msgstr "Bijv. BedrijfX"
+
+msgid "E.g., Marketing Manager"
+msgstr "Bijv. Marketing Manager"
+
+msgid "E.g., Olivia"
+msgstr "Bijv. Olivia"
+
+msgid "E.g., Rhye"
+msgstr "Bijv. Rhye"
```

Comment:
I don't know where this is used, but no Dutch person knows what Rhye is.

Reply:
Same in Danish. Don't even know the name in English, so we will change it everywhere later.

It's a placeholder text when filling out name in UI.